entrypoint.sh

```diff
@@ -12,20 +12,11 @@ if [[ -z "$PATHFINDER_URL" ]]; then
   exit 1
 fi
 
-if [[ -z "$SSO_REALM" ]]; then
-  echo "You must provide SSO_REALM environment variable" 1>&2
-  exit 1
-fi
```

Comment:
@ibolton336,

KEYCLOAK_REALM or KEYCLOAK_CLIENT_ID are legacy names which have been replaced by SSO_REALM and SSO_CLIENT.

At the moment we're using hard coded values in https://github.com/konveyor/tackle2-ui/blob/main/pkg/client/src/app/keycloak.ts.

Tackle 1 was generating a json file bundled with the app:
 https://github.com/konveyor/tackle-ui/blob/main/public/keycloak.json.template

Since those value are provided dynamically by the operator I think we  need Express to serve them.

Reply:
@gildub Do we have to keep the naming as SSO_xxxx or can they be replaced by KEYCLOAK_CLIENT_ID and KEYCLOAK_REALM ? The reason I ask is because we are getting rid of all SSO naming everywhere else.

Reply:
@fbladilo, Yes that's absolutely fine. Better have it consistent across the board